.github/workflows/build.yml

```diff
@@ -83,6 +83,14 @@ jobs:
         run: python .github/scripts/checkstyle.py
         env:
           GITHUB_WORKSPACE: ${{ github.workspace }}
+      - name: Update Floating 'cached' Tag on Successful Trunk Build
```

Comment:
I like the thorough description, but I think the "on Successful Trunk Build" qualifier is not needed. In the workflow run UI, we'll see this step skipped in all cases except on successful trunk builds :)

Reply:
Agree, have updated it!